src/knockout/templates/header.html

```diff
@@ -1,11 +1,9 @@
 ﻿<script type="text/html" id="survey-header">
-  <!-- ko if: (locTitle.koRenderedHtml().length > 0) && showTitle || hasLogo -->
+  <!-- ko if: (locTitle.koRenderedHtml().length > 0) && showTitle || hasLogo || isDesignMode -->
   <div data-bind="css: css.header">
-
       <!-- ko if: isLogoBefore -->
-      <div data-bind="css: logoClassNames">
-          <img data-bind="css: css.logoImage, attr: { src: locLogo.koRenderedHtml, width: logoWidth ? logoWidth : undefined, height: logoHeight ? logoHeight : undefined, alt: locTitle.renderedHtml }, style: { objectFit: logoFit }">
-      </div>
+      <!-- ko component: { name: getElementWrapperComponentNameByName('sv-logo-image'), params: getElementWrapperComponentDataByName('sv-logo-image') } -->
```

Comment:
The `getElementWrapperComponentNameByName` sounds weird for me. We need to discuss my comment below. I'd like to minimize `getSomethingWrapperXXXXXX` functions count.

Reply:
I agree